### DISCUSSION TITLE
How to edit the password recovery email layout ?

### DISCUSSION THREAD

**Discussion by rhonalindiefolks** [pretix/pretix#3456] [Q&A]
Hi guys, I need to change the email layout of "customer account registration, customer account email change and customer account password reset", I created plugins and used register_html_mail_renderers to register my CustomMailRenderer, it works but only the emails are related to the event.
Does anyone have any idea how to change

**Comment by raphaelm** [comment]
There's no intended way no do that currently, sorry

**Reply by rhonalindiefolks** [reply]
Thanks a lot
